frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -943,6 +997,32 @@ export const surveyLogic = kea<surveyLogicType>([
                     return SurveyQuestionBranchingType.NextQuestion
                 }
 
+                return SurveyQuestionBranchingType.ConfirmationMessage
+            },
+        ],
+        getResponseBasedBranchingDropdownValue: [
+            (s) => [s.survey],
+            (survey) => (questionIndex: number, question: RatingSurveyQuestion | MultipleSurveyQuestion, response) => {
+                if (!question.branching || !('responseValues' in question.branching)) {
+                    return SurveyQuestionBranchingType.NextQuestion
+                }
+
+                // If a value is mapped onto an integer, we're redirecting to a specific question
```

Comment:
will this work well if say my single choice options are `1`, `2`, `3`? Not sure if they are always coerced to strings

Reply:
Yes it's a text input, so always a string